SKPhotoBrowser/SKAnimator.swift

```diff
@@ -84,7 +84,9 @@ class SKAnimator: NSObject, SKPhotoBrowserAnimatorDelegate {
             let scrollView = browser.pageDisplayedAtIndex(browser.currentPageIndex) else {
                 
```

Comment:
ouch. which of the guard conditions does fail and why?

Reply:
The `browser.delegate?.viewForPhoto?(...` method if you don't implement that delegate method. Which I am not implementing on purpose  as another dismiss animation would be used then. And this animation does not fit very well with our image slider.